src/Tools/dotnet-monitor/CollectionRules/Options/CollectionRuleOptions.cs

```diff
@@ -15,23 +17,25 @@ internal sealed partial class CollectionRuleOptions
         [Display(
             ResourceType = typeof(OptionsDisplayStrings),
             Description = nameof(OptionsDisplayStrings.DisplayAttributeDescription_CollectionRuleOptions_Filters))]
-        public List<ProcessFilterDescriptor> Filters { get; } = new List<ProcessFilterDescriptor>(0);
+        public List<ProcessFilterDescriptor>? Filters { get; } = new List<ProcessFilterDescriptor>(0);
```

Comment:
The properties that are get-only and already have a default value shouldn't need to be nullable, correct?

Reply:
Just double checked, you're correct. Even if you explicitly specify `null` in the schema, it'll default to this value. I'll push a new iteration reverting  these

Reply:
Now that the PR build failed I remember why I did this -- the schema itself states that these fields are `nullable`, so they must be marked as such. This is an issue with our schema and not this PR, so I'll restore my original change and add this to the ongoign github issue.